src/main/java/de/nadirhelix/guestbook/ResourceController.java

```diff
@@ -0,0 +1,78 @@
+package de.nadirhelix.guestbook;
+
+import java.io.IOException;
+
+import javax.annotation.Resource;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import de.nadirhelix.guestbook.resources.ResourceFacade;
+
+/**
+ * This Controller handles all requests for certain resources such as .js-files or .css-files.
+ * 
+ * @author Phil
+ */
+@RestController
+@RequestMapping("/")
+public class ResourceController {
+	
+	private static Logger log = LoggerFactory.getLogger(ResourceController.class); 
```

Comment:
Warum nicht final?

Reply:
Weil ich noch kein "format on save" eingestellt in der IDE eingestellt habe. ;-)